cip-0001.md

```diff
@@ -33,23 +33,23 @@ Each CIP must have a champion -- someone who writes the CIP using the style and
 
-The CIP author may update the Draft as necessary in the [git repository](https://github.com/CounterpartyXCP/cips). Updates to drafts may also be submitted by the author as pull requests.
+The CIP author may update the draft as necessary in the [git repository](https://github.com/CounterpartyXCP/cips) via submitting a pull request. Each pull request may host discussion about those specific proposed changes to the Draft, however the CIP author and editor should stay vigiliant to avoid more general discussion on the CIP on these subsequent pull requests. Once the pull request is merged, ongoing comments being posted to that pull request should be dissuaded.
 
-Standards Track CIPs consist of two parts, a design document and a reference implementation. The CIP should be reviewed and accepted before a reference implementation is begun, unless a reference implementation will aid people in studying the CIP. Standards Track CIPs must include an implementation -- in the form of code, a patch, or a URL to same -- before it can be considered Final.
-
-CIP authors are responsible for collecting community feedback on a CIP before submitting it for review. However, wherever possible, long open-ended discussions on public forums should be avoided. Strategies to keep the discussions efficient include: setting up a separate discussion thread for the topic, having the CIP author accept private comments in the early design phases, setting up a wiki page or git repository, etc. CIP authors should use their discretion here.
+Standards Track CIPs consist of two parts, a design document and a reference implementation. The CIP should be reviewed and accepted before a reference implementation is begun, unless a reference implementation will aid people in studying the CIP. Standards Track CIPs must include an implementation -- in the form of code, a patch, or a URL to same -- before it can be considered "Final".
 
 For a CIP to be accepted it must meet certain minimum criteria. It must be a clear and complete description of the proposed enhancement. The enhancement must represent a net improvement. The proposed implementation, if applicable, must be solid and must not complicate the protocol unduly.
 
-Once a CIP has been accepted, the reference implementation must be completed. When the reference implementation is complete and accepted by the community, the status will be changed to "Final".
+To be accepted, the CIP's pull request must be approved ("ACKed") by at least two project maintainers. A project maintainer is defined as an individual that has commit access to the (counterparty-lib)[https://github.com/CounterpartyXCP/counterparty-lib] repository. If there is only one maintainer, obviously only his or her ACK is necessary.
```

Comment:
I also tweaked the definition of a maintainer a bit (require a commit in the last 6 months as adam is still listed)... let me know what you think